USING_WITH_PYRIGHT.md

```diff
@@ -15,6 +15,20 @@ Since Pyright and Pylance have different release cadences, there are many times
 
 If you are aware of other similar tools that could benefit from having the same behavior, please let us know.
 
+### Using Pyright as your diagnostics provider
+
+Pylance also has an option to allow you to run a specific version of Pyright to generate diagnostics. You might do this if you don't want to always be on the latest version of Pyright in your CI and you want your Pylance errors to match what your CI is generating.
+
+To get Pylance to run Pyight for diagnostics, set these two settings in your settings.json:


```

Comment:
What's the deal with the `^M` that GitHub is showing at the end of this line?

Reply:
I have no idea

Reply:
It renders it correctly though:

![image](https://github.com/user-attachments/assets/b05fbd72-a46a-41c9-b18f-bdd6c528f937)